client/src/components/AboutUkraine/AboutUkraine.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import {Box, Typography, styled, keyframes  } from '@mui/material';
+// import { useInView } from 'react-intersection-observer';
+
+const UkraineTextInfo = styled((props) => <Typography variant="body1" {...props} />)(({ theme }) => ({
+    [theme.breakpoints.up('tablet')]: {
+        padding: '10px',
+        fontSize: '18px',
+        lineHeight: '1.2',
```

Comment:
please delete these,  we have our default typography font styling in our theme, no need to add anything

Reply:
done